src/sentry/south_migrations/0473_auto__add_field_projectownership_auto_assignment.py

```diff
@@ -0,0 +1,1328 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
I think this migration is ok to run automatically, since the table is small. But we could just be cautious and get #ops to run it, depending on @mattrobenolt's thoughts.